planpro_importer/planpro110/reader110.py

```diff
@@ -93,14 +95,20 @@ def _get_other_uuid(_uuid, _edge):
 
             def _get_next_edge(_last_node_uuid, _second_last_node):
                 for _geo_edge in geo_edges:
-                    if _last_node_uuid in [_geo_edge.ID_GEO_Knoten_A.Wert, _geo_edge.ID_GEO_Knoten_B.Wert]:
```

Comment:
For me, personally, "last" is a confusing identifier, maybe because it's a little false friend with German "letzte:r". It could mean "previous" but also "the last one we will see in the future". Take, for example the list `1, 2, 3, 4`. We iterate until `3`. Is "last" then `2` or always `4`? I sometimes use "previous", "youngest", or "before" instead.

IIRC, "last" is being used here and there in the code. Keeping consistency would outweigh the possible confusion explained above.

Reply:
I renamed it to previous